src/languages/handlebars.js

```diff
@@ -205,7 +205,7 @@ export default function(hljs) {
   };
 
   const SUB_EXPRESSION_CONTENTS = hljs.inherit(HELPER_NAME_OR_PATH_EXPRESSION, {
-    className: 'name',
+    className: 'function',
```

Comment:
Please make sure that your PR explanation justifies (visual) changes like this and explains why they are necessary...

Reply:
can do!

In this particular case, helpers _are_ functions (and take args), and not actually just a name, like you'd have with the name of a Component invocation.